From 6e256377c31663090899f0eb647be4dac833b570 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 8 Oct 2014 14:04:52 -0400
Subject: [redhat] kabi: Adding symbol hrtimer_start (kernel/hrtimer.c)

Message-id: <984c3f8a76d9c434f1759a21d1b54db8c885d40f.1412771859.git.poros@redhat.com>
Patchwork-id: 97188
O-Subject: [RHEL7.1 PATCH 32/67] kabi: Adding symbol hrtimer_start (kernel/hrtimer.c)
Bugzilla: 1148054
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1148054
    https://bugzilla.redhat.com/show_bug.cgi?id=1148054

    Description:
    ============
    initialize sgi timers.
    Symbol used in extint_timer.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/hrtimer_start
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/hrtimer_start
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/hrtimer_start

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/hrtimer_start b/redhat/kabi/kabi-module/kabi_ppc64/hrtimer_start
new file mode 100644
index 0000000..c19d9ed
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/hrtimer_start
@@ -0,0 +1,2 @@
+#1-
+0xc3c6c9cc hrtimer_start vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/hrtimer_start b/redhat/kabi/kabi-module/kabi_s390x/hrtimer_start
new file mode 100644
index 0000000..3e2953f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/hrtimer_start
@@ -0,0 +1,2 @@
+#1-
+0xe4ba6b83 hrtimer_start vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/hrtimer_start b/redhat/kabi/kabi-module/kabi_x86_64/hrtimer_start
new file mode 100644
index 0000000..7b7894a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/hrtimer_start
@@ -0,0 +1,2 @@
+#1-
+0x6e8bf789 hrtimer_start vmlinux EXPORT_SYMBOL_GPL
-- 
1.7.1